app/controllers/openid/discovery_controller.rb

```diff
@@ -0,0 +1,45 @@
+class DiscoveryController < ApplicationController
+  def show
+    case params[:id]
+      when 'webfinger'
+        webfinger_discovery
+      when 'openid-configuration'
```

Comment:
Indent `when` as deep as `case`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.